packaging/systemd/configure-ovs.sh

```diff
@@ -737,13 +731,8 @@ if [ "$1" == "OVNKubernetes" ]; then
     fi
   fi
   convert_to_bridge "$iface" "br-ex" "phys0" "${BRIDGE_METRIC}"
-  # Check if we need to configure the second bridge
-  if [ -f "$extra_bridge_file" ] && (! nmcli connection show br-ex1 &> /dev/null || ! nmcli connection show ovs-if-phys1 &> /dev/null); then
-    interface=$(head -n 1 $extra_bridge_file)
-    convert_to_bridge "$interface" "br-ex1" "phys1" "${BRIDGE1_METRIC}"
-  fi
-  # Check if we need to remove the second bridge
-  if [ ! -f "$extra_bridge_file" ] && (nmcli connection show br-ex1 &> /dev/null || nmcli connection show ovs-if-phys1 &> /dev/null); then
+  # Remove the second bridge
+  if [ -f "$extra_bridge_file" ] && (nmcli connection show br-ex1 &> /dev/null || nmcli connection show ovs-if-phys1 &> /dev/null); then
```

Comment:
Shall we remove the check `[ -f "$extra_bridge_file" ]`?
In line 760, the second bridge and interfaces are not going to be activated even if $extra_bridge_file exist.

Reply:
This is for removing the existing extra bridge NM config files if they exist.